### ISSUE TITLE
ELB auditor can't parse ICMP ingress rule

### ISSUE THREAD

**Issue by mark-ignacio** [Netflix/security_monkey#890]
```
$ monkey find_changes -a acct
[...]
Traceback (most recent call last):
  File "/usr/local/src/security_monkey/venv/bin/monkey", line 11, in <module>
    load_entry_point('security-monkey', 'console_scripts', 'monkey')()
  File "/usr/local/src/security_monkey/security_monkey/manage.py", line 695, in main
    manager.run()
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/__init__.py", line 397, in run
    result = self.handle(sys.argv[0], sys.argv[1:])
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/__init__.py", line 376, in handle
    return handle(app, *positional_args, **kwargs)
  File "/usr/local/src/security_monkey/venv/lib/python2.7/site-packages/Flask_Script-0.6.3-py2.7.egg/flask_script/commands.py", line 145, in handle
    return self.run(*args, **kwargs)
  File "/usr/local/src/security_monkey/security_monkey/manage.py", line 80, in find_changes
    sm_find_changes(account_names, monitor_names)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 57, in find_changes
    audit_changes(accounts, monitor_names, False, debug)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 103, in audit_changes
    _audit_changes(account, monitor.auditors, send_report, debug)
  File "/usr/local/src/security_monkey/security_monkey/scheduler.py", line 135, in _audit_changes
    au.audit_objects()
  File "/usr/local/src/security_monkey/security_monkey/auditor.py", line 670, in audit_objects
    method(item)
  File "/usr/local/src/security_monkey/security_monkey/auditors/elbv2.py", line 72, in check_internet_facing
    if self._issue_matches_listeners(alb, issue):
  File "/usr/local/src/security_monkey/security_monkey/auditor.py", line 262, in _issue_matches_listeners
    from_port = to_port = int(port)
ValueError: invalid literal for int() with base 10: '0--1'
```

Print debugging showed me what this was crashing on.
```
issue.notes -- Entity: [cidr:0.0.0.0/0] Access: [ingress:icmp:0--1]
```

The attached SG seems to be:

![image](https://user-images.githubusercontent.com/4490736/33582414-01154bae-d90a-11e7-9459-54aabd5b7608.png)

**Comment by mikegrima** [comment]
Looks like we need to check for `icmp`, and handle differently.

**Comment by monkeysecurity** [comment]
Looking into this.

**Comment by monkeysecurity** [comment]
Filed #894 to fix this.

**Comment by monkeysecurity** [comment]
I've merged #894.  Issue should be fixed now.

**Comment by mark-ignacio** [comment]
Confirmed working. Thanks for the fix!
